tests/unit/test_column_noise.py

```diff
@@ -389,6 +389,9 @@ def test_miswrite_numerics(string_series):
     p_row_noise = config.row_noise_level
     p_token_noise = config.token_noise_level
     data = string_series
+    # Hack: we need to name the series something with the miswrite_numeric noising
+    # function applied to check dtypes.
+    data.name = "ssn"
```

Comment:
Why not actually give our `dummy_data` appropriate column names when we define the fixture?

Reply:
That's just an old fixture and didn't seem worth doing atm.